lib/constants/authentication_constants.dart

```diff
@@ -0,0 +1,15 @@
+//Hint Text
+const String fullNameHint = "Enter your full name";
+const String emailHint = "Enter your email";
+const String passwordHint = "Enter your password";
+const String forgotPassword = 'Forgot password?';
+
+// Buttons
+const String loginButton = "Log into your account";
+const String createAccountButtonPrimary = "Create Account";
+
+// Link sent
+const String linkSentInfoText = "Check your inbox for a confirmation link.";
+const String linkSentInfoBorder = "Link sent";
+const String linkSentFooterText = "Didn't get the email? ";
```

Comment:
I did this on purpose since we need a space character between `Didn't get the email?` and `Resend link.`. I didn't want to make another TextWidget just for the space character... unless you guys want me to!